lib/config.js

```diff
@@ -29,6 +29,8 @@ const paramsSchema = Joi.object().keys({
   errorOnRequiredAuth: Joi.boolean().optional().default(false),
   auth0Logout: Joi.boolean().optional().default(false),
   redirectUriPath: Joi.string().optional().default('/callback'),
+  loginPath: Joi.string().optional().default('/login'),
```

Comment:
I think you're assuming the `/` is always included in the value, shouldn't you validate that?

Reply:
Probably not a bad idea. I'm using the same convention as the callback but it's probably good to do for all 3. Good catch.

#21